app/serializers/reviewable_action_serializer.rb

```diff
@@ -8,8 +8,7 @@ class ReviewableActionSerializer < ApplicationSerializer
              :confirm_message,
              :description,
              :client_action,
-             :require_reject_reason,
-             :custom_modal
```

Comment:
I'm not 100% sure but is this a row in the database and if so do we need to clean this up?

Reply:
Nah no DB row, it was just an attribute @tgxworld